### DISCUSSION TITLE
trying to train but got some error (newbie)

### DISCUSSION THREAD

**Discussion by wtmayhem** [acids-ircam/RAVE#211] [Q&A]
I am new to here, currently using an m1 mac, after spending sometime I finallly got it installed,
so after setup the dataset, I can see it generated the file named data.mlb and lock.mlb plus metadata.yami.
after run the trainning code, it give me this error code "Training on mac is not available yet. Use --gpu -1 to train on CPU (not recommended). "

**Comment by caillonantoine** [comment]
Hi ! You shouldn't train on a M1, training is gonna be super slow ! I recommend you use a computer with a cuda / rocm graphic card or use colab :)

**Comment by asuioplaksjsjskjj** [comment]
damn
